src/parser/monk/mistweaver/modules/spells/InvokeChiJi.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+
+import SpellIcon from 'common/SpellIcon';
+import ItemHealingDone from 'interface/ItemHealingDone';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+
+class InvokeChiJi extends Analyzer {
+  healing = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.INVOKE_CHIJI_THE_RED_CRANE_TALENT.id);
```

Comment:
Right after this you should add 
`if(!this.active){
return;
}
`
To prevent adding unneeded action listeners